src/main/react/clients/UserClient.js

```diff
@@ -0,0 +1,33 @@
+import {
+  checkResponse,
+  validateResponse,
+} from "@flock-community/flock-eco-core/src/main/react/clients/utils.ts";
+
+const path = "/login";
+
+const getStatus = () => {
+  const opts = {
```

Comment:
Afaik `get` is the default so when getting you don't need to send an options object with it